setup.sh

```diff
@@ -21,6 +21,7 @@ sudo apt-get install python-psycopg2 --yes
 sudo apt-get install python-gobject --yes
 sudo apt-get install python-pycurl --yes
 sudo apt-get install postgresql-9.3-postgis-2.1 --yes
+sudo apt-get install landscape-client --yes
```

Comment:
I agree, it is weird. We should delve into the reqs at some point to determine if they're all used. Probably when we redo the server